### PR TITLE
Add some updates to servicemesh.es.

### PR DESCRIPTION
**Pull request by craigbox**
Some updates to servicemesh.es.

- remove Service Mesh Interface as the project is archived
- update Gateway API copy and logo
- bump Istio version and information about Ambient Mesh
- update Cilium description to separate Cilium CNI from Cilium Service Mesh
- note Consul's license change to BSL

### CODE REVIEW

**Review by MarcoSteinke — APPROVED** [review]

File: _includes/comparison.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>Apache License 2.0</td>
         <td>Apache License 2.0</td>
         <td>Closed Source</td>
-        <td>Mozilla License</td>
+        <td>Business Source License</td>
```
Comment (MarcoSteinke) [inline _includes/comparison.html:32]: ✅

File: _includes/comparison.html [hunk 2]
```diff
@@ -46,7 +46,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
       </tr>
       <tr>
         <th scope="row">Service Proxy</th>
-        <td><a href="https://www.envoyproxy.io" target="_blank">Envoy</a>, <a href="https://istio.io/latest/blog/2021/proxyless-grpc/proxyless" target="_blank">proxyless for gRPC (experimental)</a></td>
+        <td><a href="https://www.envoyproxy.io" target="_blank">Envoy</a>, <a href="https://istio.io/latest/blog/2022/introducing-ambient-mesh/" target="_blank">Ambient Mesh (alpha)</a></td>
```
Comment (MarcoSteinke) [inline _includes/comparison.html:49]: ✅

File: _includes/comparison.html [hunk 3]
```diff
@@ -56,12 +56,12 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
       </tr>
       <tr>
         <th scope="row">Ingress Controller</th>
-        <td><a href="https://istio.io/docs/tasks/traffic-management/ingress/" target="_blank">Envoy / Own Concept</a>, support for <a href="https://gateway-api.sigs.k8s.io/" target="_blank">Kubernetes Gateway API</a></td>
-        <td>any</td>
+        <td><a href="https://istio.io/docs/tasks/traffic-management/ingress/" target="_blank">Envoy</a>, support for <a href="https://gateway-api.sigs.k8s.io/" target="_blank">Kubernetes Gateway API</a></td>
```
Comment (MarcoSteinke) [inline _includes/comparison.html:59]: ✅

File: _includes/comparison.html [hunk 4]
```diff
@@ -236,53 +236,6 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://docs.cilium.io/en/latest/installation/clustermesh/clustermesh/#gs-clustermesh" target="_blank">yes</a></td>
       </tr>
     </tbody>
-    <thead style="background-color: white; color: #ff4d67">
```
Comment (MarcoSteinke) [inline _includes/comparison.html:239]: I guess this was changed due to the transition from SMI to Gateway API ? @craigbox

File: _includes/implementations.html [hunk 5]
```diff
@@ -92,7 +92,7 @@ <h2 class="list-teaser__headline">Kuma</h2>
           </div>
           <h2 class="list-teaser__headline">Cilium</h2>
           <p class="list-teaser__text">
-            Cilium existed long before the term Service Mesh even came up, providing security, networking and observability features efficiently with eBPF instead of sidecars.
+            Cilium is a network plugin for Kubernetes using the eBPF features of the Linux kernel. Recent releases also supports some mesh features, with or without sidecars.
```
Comment (MarcoSteinke) [inline _includes/implementations.html]: ✅

File: _includes/implementations.html [hunk 6]
```diff
@@ -119,7 +119,7 @@ <h2 class="list-teaser__headline">Gloo Mesh</h2>
           </div>
           <h2 class="list-teaser__headline">Gateway API</h2>
           <p class="list-teaser__text">
-            Gateway API is a collection of resources centered around Gateway resources (which represent the underlying network gateways / proxy servers) to enable robust Kubernetes service networking through expressive, extensible and role-oriented interfaces that are implemented by many vendors and have broad industry support.
+            Gateway API is a collection of resources, initially designed to replace the Kubernetes Ingress API. The expressive, extensible and role-oriented interfaces are now extending into service mesh use cases, with broad industry support.
```
Comment (MarcoSteinke) [inline _includes/implementations.html:122]: ✅

File: _includes/comparison.html [hunk 7]
```diff
@@ -112,7 +112,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
             <td>Consul uses its own internal storage, and does not on rely Kubernetes for persistent storage.</td>
             <td>Traefik Mesh currently does not support transparent TLS encryption.</td>
             <td>Kuma is possibly the most flexible service mesh. Teams should thoroughly consider whether their project can handle the complexity involved.</td>
-            <td>To enable the same feature set as Service Meshes with sidecars, a lot of maunal configuration is used.</td>
+            <td>Any L7 use cases require the use of an Envoy proxy, which is shared between all workloads on the node and not a recommended security configuration.</td>
```
Comment (MarcoSteinke) [inline _includes/comparison.html:115]: ✅

File: _includes/comparison.html (hunk 4 above)
Comment (craigbox) [reply _includes/comparison.html:239]: Yes; there's no point in listing compatibility with dead APIs.  I haven't proposed a section for compatibility with Gateway API/GAMMA APIs as (a) they're not done yet (b) they're tracked [here](https://gateway-api.sigs.k8s.io/implementations/).

Comment (MarcoSteinke) [reply _includes/comparison.html:239]: Thank you for your additions and your support :) I will merge your contributions.

**Pull request merged**